.travis.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Do we need this?

Reply:
Standard YAML syntax, but Travis will build it without it. I'm used to it b/c of Ansible